nibabel/testing/__init__.py

```diff
@@ -21,8 +21,18 @@
 try:
     from numpy.testing import dec
     skipif = dec.skipif
+    slow = dec.slow
 except ImportError:
     from numpy.testing.decorators import skipif
+
+    # Recent (1.2) versions of numpy have this decorator
```

Comment:
We have a minimum version of 1.7.1. I don't think this is guard is necessary anymore.